### PR TITLE
Users can change their password on account details page

### PR DESCRIPTION
**Pull request by hedgehog1833**

### CODE REVIEW

**Comment by hedgehog1833** [comment]
[Change password](https://trello.com/c/B758Jj1N/259-change-password)

**Comment by DanielW1987** [comment]
Don't know why I approve here :thinking: 
would be happy if you could still address my comments despite Approval.

File: webapp/src/main/resources/static/ts/src/service/account-details-render-service.ts [hunk 1]
```diff
@@ -39,4 +42,84 @@ export class AccountDetailsRenderService {
...
+        if (oldPasswordMathMessage != null) {
+            const listItem = document.createElement("li") as HTMLLIElement;
+            listItem.textContent = "Old password does not match.";
+            messages.appendChild(listItem);
+        }
+
+        messagesSpan.innerHTML = messages.innerHTML;
+        errorMessageAlert.appendChild(closeButton);
+        errorMessageAlert.appendChild(messagesSpan);
+    }
```
[the remark this answers is not in the document]
Comment (hedgehog1833) [reply webapp/src/main/resources/static/ts/src/service/account-details-render-service.ts]: Okay I reworked all of the message formatting a bit, I'm still not a 100% satisfied. Especially when there are multiple errors with the password it look pretty dumb.. But in my opinion it's GEFN 😉 What do you think?
I also removed the under construction banner as now all button and so on are working.

**Comment by DanielW1987** [comment]
Yes, it doesn't look really nice. The messages were also never intended to be displayed in the frontend. 

I'm thinking about adding messages for the frontend for a later PR.

For now it can be merged.

**Pull request merged**